processor/k8sprocessor/processor.go

```diff
@@ -102,13 +112,30 @@ func (kp *kubernetesprocessor) ConsumeTraceData(ctx context.Context, td consumer
 	if podIP == "" && td.SourceFormat == sourceFormatJaeger {
 		if td.Node != nil {
 			podIP = td.Node.Attributes[ipLabelName]
+			kp.debugIPMaybe("Pod IP in Node", podIP)
+		}
+	}
+
+	// If this was passed using Zipkin format, the information used for tagging might be present
+	// in each span attribute and should be tagged not on a resource, but rather span level
+	if podIP == "" && td.SourceFormat == sourceFormatZipkin {
```

Comment:
There is a section dealing with Jaeger input format just above that so I followed the pattern. :) 

My understanding is that for Jaeger, the IP is found in `Node` attributes section rather than `Resource` labels, hence the separate handling

For Zipkin, the IP address payload comes in each individual span, hence the flow is different. Zipkin receiver has a piece which attempts to extract common nodes and group the spans by local and remote endpoints: https://github.com/open-telemetry/opentelemetry-collector/blob/master/receiver/zipkinreceiver/trace_receiver.go#L203-L233 

Not sure if that's the best way to go, but perhaps some attributes could be moved from Span level to Node level and being used in grouping? That would effect in nodes being grouped by not only local/remote endpoints but also those attributes. I believe this will almost always yield the same result, but I wasn't confident enough to use such approach. What do you think?